TRACK_README.md

```diff
@@ -1,11 +1,58 @@
 # Exercism {{LANGUAGE}} Track
 
+![build status](https://travis-ci.org/exercism/xslug.svg?branch=master)
+
 Exercism exercises in {{LANGUAGE}}.
 
-## TODO
+## Setup
+
+The simplest way to install {{LANGUAGE}} is ...
+
+## Contributing
+
+Thank you so much for contributing! :tada:
+
+Please start by reading the general Exercism [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
+
+We welcome pull requests that provide fixes and improvements to existing exercises. If you're unsure, then go ahead and open a GitHub issue, and we'll discuss the change.
+
+Please keep the following in mind:
+
+- Pull requests should be focused on a single exercise, issue, or change.
+
+- We welcome changes to code style, and wording. Please open a separate PR for these changes if possible.
+
+- Please open an issue before creating a PR that makes significant (breaking) changes to an existing exercise or makes changes across many exercises. It is best to discuss these changes before doing the work.
+
+- Follow the coding standards for {{LANGUAGE}}. (If there is a formatter for the track's language, add instructions for using it here.)
+
+- Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
+
+- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+
+- Exercises should use only the {{LANGUAGE}} core libraries.
+
+- Please do not add a README or README.md file to the exercise directory. The READMEs are constructed using shared metadata, which lives in the
+[exercism/x-common](https://github.com/exercism/x-common) repository.
```

Comment:
We can link to https://github.com/exercism/exercism.io/blob/master/docs/fixing-exercise-readmes.md here